src/component/Edit/Editproperty.jsx

```diff
@@ -0,0 +1,239 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import Selectoption from "../../Dashboard/Selectoption";
+import axios from "axios";
+
+const url = "http://localhost:3006/v1//property/:userId";
+
+const Editproperty = () => {
+ console.log(useParams())
+ const {id} = useParams()
+
+ console.log(id)
+ const getToken = JSON.parse(localStorage.getItem('data'));
+
+ let config = {
+   "headers": {
+     'Authentication': getToken
+   }
+ }
+  const defaultFormData = {
+    title: "",
+    address: "",
+    purpose: "",
+    property_type: "",
+    // city: "",
+    image_url: "",
+    no_of_beds: "",
+    no_of_baths: "",
+    garages: "",
+    land_size: "",
+    locality: "",
+    neighbourhood: "",
+    description: "",
+  };
+  const [formData, setFormData] = useState(defaultFormData);
+
+  const handleChange = (e) => {
+    const { value, name } = e.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    console.log(formData)
+
+    try {
+      const response = await axios.put(`http://localhost:3006/v1//property/${id}`, { ...formData }, config);
```

Comment:
You shouldn't hardcode your API URL. It should be dynamic based on the environment you are on. When you deploy this code online, your API URL will no longer be localhost:3006.

Reply:
worked on boss